CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Changelog
 
+## [1.62](https://github.com/brave/brave-ios/releases/tag/v1.62)
+
+### Web3
+
+- Implemented Safe Sign for CoW Swaps. [(#8114)](https://github.com/brave/brave-ios/issues/8114)
+- Added support for fetching unknown tokens from non-mainnet transactions. [(#8529)](https://github.com/brave/brave-ios/issues/8529)
+- Added “Creating Wallet” step to Wallet onboarding. [(#8345)](https://github.com/brave/brave-ios/issues/8345)
+- Added “Accounts” and “Transactions” tab to the asset detail view. [(#8137)](https://github.com/brave/brave-ios/issues/8137)
+- Updated Line chart color to match the new Brave Wallet design on Portfolio and Asset details screen. [(#8136)](https://github.com/brave/brave-ios/issues/8136)
+- Updated account blockies to be square similar to desktop. [(#8375)](https://github.com/brave/brave-ios/issues/8375)
+- Migrated Solana Swaps to use Jupiter Swap API v6. [(#8603)](https://github.com/brave/brave-ios/issues/8603)
+- Fixed the approve screen not showing the confirmation when approve limit is lower than transaction value. [(#8721)](https://github.com/brave/brave-ios/issues/8721)
+- Fixed portfolio filter from opening intermittently when scrolling up and down the portfolio asset list. [(#8709)](https://github.com/brave/brave-ios/issues/8709)
+- Fixed UI issues in Brave Wallet resulting from NFT GIFs failing to load. [(#8560)](https://github.com/brave/brave-ios/issues/8560)
+- Fixed Wallet onboarding flow when initiated on two windows in split view on iPad. [(#8425)](https://github.com/brave/brave-ios/issues/8425)
+
+### General
+
+- Added Global Privacy Control toggle under “Brave Shields & Privacy.” [(#8556)](https://github.com/brave/brave-ios/issues/8556)
```

Comment:
nit: lets add `Global Privacy Control` into quotes. Looks cleaner/easier to read. Example:

```
- Added "Global Privacy Control" toggle under “Brave Shields & Privacy.” [(#8556)](https://github.com/brave/brave-ios/issues/8556)
```